### PR TITLE
Fix manifest merger error when targetApi is present in the AndroidManifest.

### PR DESCRIPTION
**Pull request by raviagarwal7**
Patch picked from - https://android.googlesource.com/platform/tools/base/+/5c6d992f405b0d83f3c10e4d5f6d961bef50221c%5E%21/build-system/manifest-merger/src/main/java/com/android/manifmerger

Google Issue tracker - https://issuetracker.google.com/issues/114358542.
Fix in bazel - https://github.com/bazelbuild/bazel/commit/59327306bf7bb6ecc28d8b52c8bbf343361293bf

### CODE REVIEW

**Comment by styurin** [comment]
I have changes for this issue in the review already

File: third-party/java/aosp/changes.patch [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
++import java.util.Locale;
+ import org.w3c.dom.Attr;
+ import org.w3c.dom.Element;
+ import org.w3c.dom.NamedNodeMap;
+@@ -126,7 +127,7 @@ public class XmlElement extends OrphanXmlElement {
+                     } catch (IllegalArgumentException e) {
+                         try {
+                             // is this another tool's operation type that we do not care about.
+-                            OtherOperationType.valueOf(instruction);
++                            OtherOperationType.valueOf(instruction.toLowerCase(Locale.ROOT));
```
Comment (styurin) [inline third-party/java/aosp/changes.patch]: Why do we need this?

Comment (styurin) [reply third-party/java/aosp/changes.patch]: I mean the change in bazel only deal with `OtherOperationType`

Comment (raviagarwal7) [reply third-party/java/aosp/changes.patch]: the change in later version of manifest merger does this to account for all kind of targetApi usages. 
https://android.googlesource.com/platform/tools/base/+/5c6d992f405b0d83f3c10e4d5f6d961bef50221c%5E%21/build-system/manifest-merger/src/main/java/com/android/manifmerger

**Comment by raviagarwal7** [comment]
> I have changes for this issue in the review already

cool, I will close this.

**Pull request closed without merging**